source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -37,6 +37,9 @@ void SplitRequestBase::updateStats(const bool success) {
   } else {
     command_stats_.error_.inc();
   }
+  std::chrono::milliseconds latency = std::chrono::duration_cast<std::chrono::milliseconds>(
```

Comment:
I see that there's a Timespan class in Envoy.Stats that does the same thing.  Maybe we should use that so that it's more consistent.

Reply:
OK. Changed to use `Timespan` class. PTAL.